app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -157,6 +157,24 @@ class Contact extends React.Component {
                       <option>Afternoon</option>
                       <option>Evening</option>
                     </Form.Control>
+                    <Form.Row className="pt-2">
+                      `{' '}
+                      <Form.Group as={Col} md="auto">
+                        Morning:
+                        <br />
+                        Afternoon:
+                        <br />
+                        Evening:
+                      </Form.Group>
+                      <Form.Group as={Col} md="auto">
+                        <span className="font-weight-light">Between 8:00 and 12:00 in monitoree&apos;s timezone</span>
+                        <br />
+                        <span className="font-weight-light">Between 12:00 and 16:00 in monitoree&apos;s timezone</span>
+                        <br />
+                        <span className="font-weight-light">Between 16:00 and 20:00 in monitoree&apos;s timezone</span>
+                      </Form.Group>
+                      `
```

Comment:
What's going on with these tic marks?

Reply:
The Apostrophes? It's the monitoree's timezone so it would be possessive right?

Reply:
No the tick marks on line 161 and 176

Reply:
Oh.. missed those. yeah idk how those got there. Deleted.